c/tskit/tables.h

```diff
@@ -842,6 +842,30 @@ int tsk_individual_table_clear(tsk_individual_table_t *self);
 */
 int tsk_individual_table_truncate(tsk_individual_table_t *self, tsk_size_t num_rows);
 
+/**
+@brief Extends a table by appending rows copied from another table.
+
+@rst
+Rows from ``other``, specified by row index, are appended to the end of  ``self``.
+Row indexes can be repeated. If ``row_indexes`` is NULL all rows are copied.
+``other`` is not modified. Note that metadata is copied as-is and is not checked
+for compatibility with any existing schema on ``self``.
+@endrst
+
+@param self A pointer to a tsk_individual_table_t object where rows are to be added.
+@param other A pointer to a tsk_individual_table_t object where rows are copied from.
+@param num_row_indexes The number of entries to use from ``row_indexes``, ignored if
```

Comment:
I was thinking more like: 
> @param num_rows The number of rows from other to append to this table.

This seems more flexible and useful: if we want to get the first 10 rows, say, we just pass in ``.., 10, NULL)``.